content/200-orm/200-prisma-client/100-queries/037-relation-queries.mdx

```diff
@@ -24,6 +24,63 @@ Nested reads allow you to read related data from multiple tables in your databas
 - Use [`include`](/orm/reference/prisma-client-reference#include) to include related records, such as a user's posts or profile, in the query response.
 - Use a nested [`select`](/orm/reference/prisma-client-reference#select) to include specific fields from a related record. You can also nest `select` inside an `include`.
 
+### Relation load strategies (Preview)
+
+Since [5.9.0](https://github.com/prisma/prisma/releases/tag/5.9.0), you can decide on a per-query-level _how_ you want Prisma Client to execute a relation query (i.e. what _load strategy_ should be applied) via the `relationLoadStrategy` option.
+
+Because the `relationLoadStrategy` option is currently in Preview, you need to enable it via the `relationJoins` preview feature flag in your Prisma schema file:
+
+```prisma file=schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["relationJoins"]
+}
+```
+
+Prisma Client supports two load strategies for relations:
+
+- `join`: Uses a database-level `LATERAL JOIN` and fetches all data with a single query to the database.
+- `query` (default): Sends multiple queries to the database (one per table) and joins them on the application level.
```

Comment:
`join` is the default when the preview feature is enabled.

Reply:
Ah thanks! I thought this would only be changed when moved to GA 👍  will correct this in the text!